src/constants.js

```diff
@@ -22,6 +22,11 @@ b.style.float = 'right';
 document.querySelector('body').append(b);
 `;
 
+const DEC_128_PREFIX = `
+  // Decimal128 allows for 34 digits of significand
+  Decimal.set({precision: 34});
+`;
+
```

Comment:
Side point, but I think we still need to set the Decimal precision before the transformed code is run even when loading everything else in a script because it needs to set the environment.

Reply:
I'm not sure I get why this is? Maybe it will become clear if I try to run it.

(My motivation for changing this in the previous PR is that it's annoying to have JS code contained inside strings where it can't be linted.)

Reply:
I tried it in #14, and it works (see the commit "Move Decimal prefix into patches.js")